languages.go

```diff
@@ -8,4 +8,5 @@ const (
 	NoLang    LangType = ""
 	GoLang             = "go"
 	SwiftLang          = "swift"
+	KotlinLang		   = "kotlin"
```

Comment:
Can you recommit when using gofmt? The indentation is off.

Reply:
sure